### ISSUE TITLE
Deprecation warning for legacy importlib api

### ISSUE THREAD

**Issue by quimey** [IdentityPython/pysaml2#832]
<!--- Provide a general summary of the issue in the Title above -->

## Code Version
I've found the issues in the latest version (7.0.1)
```
$ pip freeze | grep -e saml -e importlib
django-saml2-auth @ git+git://github.com/Avature/django-saml2-auth@1b7f14edc7f813366a8d55ffb36bc39732c9748c
importlib-resources==5.3.0
pysaml2==7.0.1
```

## Expected Behavior
No deprecation warnings.

## Current Behavior
Deprecation warnings.

## Possible Solution
Use new api  https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy

## Steps to Reproduce
Use the library with a current installation of importlib.
```
    from importlib_resources import path as _resource_path                                                                                                                                                 

    from xmlschema import XMLSchema as _XMLSchema                                                                                                  
    from xmlschema.exceptions import XMLSchemaException as XMLSchemaError
                                
    import saml2.data.schemas as _data_schemas
                                                          
                                          
    def _create_xml_schema_validator(source, **kwargs):
        kwargs = {                          
            **kwargs,                               
            "validation": "strict",                      
            "locations": _locations,
            "base_url": source,                                       
            "allow": "sandbox",                                                                                                                                                                            
            "use_fallback": False,
        }                                                  
        return _XMLSchema(source, **kwargs)
                        
                       
>   with _resource_path(_data_schemas, "xml.xsd") as fp:
                                            
.tox/py38/lib/python3.8/site-packages/saml2/xml/schema/__init__.py:21:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

args = (<module 'saml2.data.schemas' from '/home/quimey/trabajo/itt/.tox/py38/lib/python3.8/site-packages/saml2/data/schemas/__init__.py'>, 'xml.xsd'), kwargs = {}
                                                                                                                                                                                                           
    @functools.wraps(func)
    def wrapper(*args, **kwargs):                          
>       warnings.warn(                                                                                                                                                                   
            f"{func.__name__} is deprecated. Use files() instead. "
            "Refer to https://importlib-resources.readthedocs.io"                                                                                                                                           
            "/en/latest/using.html#migrating-from-legacy for migration advice.",
            DeprecationWarning,                                 
            stacklevel=2,                                         
        )                                                                                                  
E       DeprecationWarning: path is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
```

**Comment by c00kiemon5ter** [comment]
closed by 5583f16

**Comment by invalidusrname** [comment]
mind cutting a release for this? I'm seeing these deprecation warnings too :smile:
